docker-compose.yml

```diff
@@ -10,24 +10,39 @@ services:
     volumes:
       - database:/var/lib/postgresql/data
 
-  app:
+  redis:
+    env_file: .env
+    expose:
+      - 6379
+    image: redis:5.0.6-alpine
```

Comment:
Not sure if it makes a difference but AWS redis engine I believe only goes up to `5.0.5` so maybe we can change the image name to match that.

Reply:
Ah, Docker Hub doesn't have an official 5.0.5 redis: https://hub.docker.com/_/redis/

(There is one around, but it's of unknown provenance https://hub.docker.com/search?q=redis%3A5.0.5&type=image)

OK to stick with 5.0.6?